client/data/sites/use-site-excerpts-query.ts

```diff
@@ -29,6 +29,16 @@ const fetchSites = (
 	} );
 };
 
+export const queryKeyForAllSitesWithThemeSlug = [
+	USE_SITE_EXCERPTS_QUERY_KEY,
+	SITE_EXCERPT_REQUEST_FIELDS,
+	SITE_EXCERPT_REQUEST_OPTIONS,
+	[],
+	'all',
+	[],
+	[ 'theme_slug' ],
+];
+
```

Comment:
Closing the loop: I mentioned on Slack that I think we should replace this with a `getUseSiteExcerptsQueryKey` function. That function should be used internally in `useSiteExcerptsQuery` and when invalidating the query.